libqtile/resources/status_notifier/dbusmenu.py

```diff
@@ -0,0 +1,71 @@
+SCHEMA_DBUS_MENU = """
```

Comment:
As this is no longer used in this version, could it also be moved to the other repo?

Reply:
Dammit. Yes.